_config.yml

```diff
@@ -2,6 +2,7 @@ name: "LLNL Software Portal"
 author: "LLNL Open Source <open-source@llnl.gov>"
 title: LLNL Software Portal
 description: "Lawrence Livermore National Laboratory Software Portal"
+remote_theme: LLNL/Jekyll-LLNL-Theme
```

Comment:
Do we not also need the plugin listed in this file?

Reply:
The README for the [LLNL remote theme](https://github.com/LLNL/Jekyll-LLNL-Theme) doesn't mention the Jekyll remote theme plugin. I'll make a new commit to incorporate the plugin as you did with the dev.llnl.gov site

Reply:
ok, I mean if it worked, then I wasn't going to worry about it, but I'd thought it needed to be there.